Opt-out updated bootimage for GCP and AWS (#93065)
openshift/openshift-docs: modules/mco-update-boot-images-configuring.adoc

:_mod-docs-content-type: PROCEDURE
[id="mco-update-boot-images-configuring_{context}"]
= Re-enabling updated boot images

> By default, for {gcp-first} and {aws-first} clusters, by default the Machine Config Operator (MCO) manages and updates the boot image in the machine sets in your cluster. This means that the MCO updates the boot images whenever you update your cluster.

Reviewer: This is worded confusingly to me, so removed a few words.
```suggestion
By default, for {gcp-first} and {aws-first} clusters, the Machine Config Operator (MCO) manages and updates the boot image in the machine sets in your cluster. This means that the MCO updates the boot images whenever you upgrade your cluster.
```

If you disabled this default behavior, where the boot images are not updated, you can revert to the default behavior by editing the `MachineConfiguration` object.  

> Re-enabling the default behavior updates the boot image to the current {product-title} version. If the cluster is again updated to a new {product-title} version in the future, the boot image is updated again. New nodes created after enabling the feature use the updated boot image. Existing nodes are unchanged, retaining their current boot image.

Reviewer: ```suggestion
Re-enabling the default behavior updates the boot image to the current {product-title} version. If the cluster is again updated to a new {product-title} version in the future, the boot image is updated again. New nodes created after enabling the feature use the updated boot image. This feature has no effect on existing nodes.
```
Minor rephrase for clarifying existing node behavior.

> Re-enabling the default behavior after some nodes had been created does not affect existing nodes or machine sets. The machine sets retain the boot image version that was present when the feature was disabled. Any existing nodes retain their current boot image. However, the cluster boot image is updated again if the cluster is upgraded to a new {product-title} version in the future, and new nodes created after that point use the updated boot image.

Reviewer: This is not entirely accurate. If you re-enable the feature at a later time, the machinesets will be updated to match the boot image the current OCP version. If the boot images already match the current OCP version, no update will take place. I just want to make it clear that it doesn't just happen on an upgrade; For example, you could have disabled the feature say in 4.19, upgraded to 4.21, and then re-enabled the feature - the MCO will immediately update the boot image to 4.21.

And as stated earlier, boot images have no effect on existing nodes.  Just a suggestion for the text, probably could clean it up a bit more: 
```suggestion
Re-enabling the default behavior will update the cluster's boot images to match the current OCP version. If the cluster is upgraded to a new {product-title} version in the future, the boot images will be updated again. New nodes created after enabling the feature will use the updated boot image.
```
